versions.tf

```diff
@@ -6,7 +6,7 @@ terraform {
     }
     metal = {
       source  = "equinix/metal"
-      version = "~> 3.0"
+      version = "3.2.0-alpha.5"
```

Comment:
prior to the next tagged release of this module, this version should be pinned to a non-alpha release

Reply:
not a blocker for merge, imho